drivers/mm/mm_drv_intel_adsp_mtl_tlb.c

```diff
@@ -20,7 +20,7 @@
  * are untouched.
  */
 
-#define DT_DRV_COMPAT intel_adsp_tlb
+#define DT_DRV_COMPAT DT_DRV_COMPAT_ADSP_TLB
```

Comment:
why this change ?

Reply:
because the string name "intel_adsp_tlb" is used twice, in .h and in .c files, so better to keep the string in a macro 
If you disagree I'll change it


mm_drv_intel_adsp_mtl_tlb.h:
/* device id from the device tree */
#define DT_DRV_COMPAT_ADSP_TLB intel_adsp_tlb

/* size of TLB table */
#define TLB_SIZE DT_REG_SIZE_BY_IDX(DT_INST(0, **DT_DRV_COMPAT_ADSP_TLB**), 0)


mm_drv_intel_adsp_mtl_tlb.c:
#define DT_DRV_COMPAT **DT_DRV_COMPAT_ADSP_TLB**